Khoj.spec

```diff
@@ -19,6 +19,9 @@ datas += copy_metadata('filelock')
 datas += copy_metadata('numpy')
 datas += copy_metadata('tokenizers')
 datas += copy_metadata('pillow')
+datas += copy_metadata('huggingface_hub')
+datas += copy_metadata('safetensors')
+datas += copy_metadata('pyyaml')
```

Comment:
So once we `copy_metadata` of these dependencies, the khoj exe for windows runs fine now?

Reply:
Correct - after adding these lines, both Windows and Linux desktop builds were unblocked.